### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Smeds**
Started on writing down information about hydra and what it's supposed to supply.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+#### [hydra-genetics/alignment](https://github.com/hydra-genetics/alignment)
+Snakemake module containing processing steps that are be performed during sequence alignment.
+
+#### [hydra-genetics/snv_indels](https://github.com/hydra-genetics/snv_indels)
+Collection of rules used to call snv and small indels
+
+#### [hydra-genetics/annotation](https://github.com/hydra-genetics/annotation)
+Collections of rule to annotate vcf files.
+
+#### [hydra-genetics/cv_sv](https://github.com/hydra-genetics/cv_sv)
```
Comment (jonca79) [inline README.md]: Wrong linkand name: should be https://github.com/hydra-genetics/cnv_sv

File: README.md [hunk 2]
```diff
@@ -1,2 +1,68 @@
...
+Collection of rules call  structural variants 
+
+#### [hydra-genetics/biomarkers](https://github.com/hydra-genetics/biomarkers)
+Collection of rules used to calculate biomarkers like Msi, TMB and HRD.
```
Comment (jonca79) [inline README.md]: MSI

File: README.md [hunk 3]
```diff
@@ -1,2 +1,68 @@
...
+## Wiki
+Please visit our [wiki](https://github.com/hydra-genetics/.github/wiki/Welcome-to-hydra-genetics) for more information about using hydra-genetics.
+
+## Repositories 
+The current repositories can be divided into the following sections.
+
+### Tools and examples
+
+#### [hydra-genetics/tools](https://github.com/hydra-genetics/tools)
+Command line interface to create new modules/pipelines or adding a new rule to a existing project. Provides libraries used to make it easier for people not used to [pandas](https://pandas.pydata.org/) to extract information from samples and units dataframes, these dataframes are generated from [units.tsv](https://github.com/hydra-genetics/prealignment/blob/develop/workflow/schemas/units.schema.yaml) and [samples.tsv](https://github.com/hydra-genetics/prealignment/blob/develop/workflow/schemas/samples.schema.yaml) files  and used as input for hydra-genetics.
```
Comment (jonca79) [inline README.md]: Feels a bit misleading to link to units.tsv and then get a schema. Instead link to example and the schema.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,68 @@
...
+Snakemake module containing processing steps that are be performed before sequence alignment.
+
+#### [hydra-genetics/alignment](https://github.com/hydra-genetics/alignment)
+Snakemake module containing processing steps that are be performed during sequence alignment.
+
+#### [hydra-genetics/snv_indels](https://github.com/hydra-genetics/snv_indels)
+Collection of rules used to call snv and small indels
+
+#### [hydra-genetics/annotation](https://github.com/hydra-genetics/annotation)
+Collections of rule to annotate vcf files.
```
Comment (jonca79) [inline README.md]: Collection of rules

File: README.md [hunk 5]
```diff
@@ -1,2 +1,68 @@
...
+
+#### [hydra-genetics/cv_sv](https://github.com/hydra-genetics/cv_sv)
+Collection of rules call  structural variants
```
Comment (jonca79) [inline README.md]: Collection of rules used to call structural variants

File: README.md [hunk 6]
```diff
@@ -1,2 +1,68 @@
...
+Collections of rule to annotate vcf files.
+
+#### [hydra-genetics/cv_sv](https://github.com/hydra-genetics/cv_sv)
+Collection of rules call  structural variants 
+
+#### [hydra-genetics/biomarkers](https://github.com/hydra-genetics/biomarkers)
+Collection of rules used to calculate biomarkers like Msi, TMB and HRD.
+
+#### [hydra-genetics/qc](https://github.com/hydra-genetics/qc)
+Collection of rules for performing calculating QC and generating reports.
```
Comment (jonca79) [inline README.md]: Collection of rules performing QC and generating reports.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,68 @@
...
+
+#### [hydra-genetics/misc](https://github.com/hydra-genetics/misc)
+
+#### [hydra-genetics/references](https://github.com/hydra-genetics/references)
+Collection of rules used to create references, PoN, and background filters.
```
Comment (jonca79) [inline README.md]: panel of normals

File: README.md (hunk 1 above)
Comment (Smeds) [reply README.md]:
```suggestion
#### [hydra-genetics/cv_sv](https://github.com/hydra-genetics/cnv_sv)
```

**Comment by Smeds** [comment]
@cl2014git kan du också ta en titt?

**Review by jonca79 — APPROVED** [review]

**Pull request merged**